docs/configuration.md

```diff
@@ -1664,6 +1664,14 @@ Apart from these, the following properties are also available, and may be useful
         </td>
     </tr>
     <tr>
+        <td><code>spark.ssl.<particular protocol>.port</code></td>
```

Comment:
It has a reason for this.
If an user set spark.ssl.port, the port of history server, SparkUI, MasterWebUI, and WorkerWebUI will collapse as I mentioned at the beginning of this PR.
It is because WebUI.scala is inheried by HistoryServer.scala, SparkUI, MasterWebUI, and WorkerWebUI.
How do you think about it?


> I created a pull request, but there is a problem.
> Port should be separated for each particular process such as history server and spark 
> ui.
> But user can configure spark.ssl.port with this code and when configure it, each 
> process will try to create connections with same port and it doesn't work good.
> Because of structure of existing function, it has impact to other functions to solve the
> problem.
> So there are three answer about it.
>
> Use this fix code so user can configure a specific secure port, but spark.ssl.port doesn't > work good.
> Don't change and close issue.
> We should be able to configure a specific secure port completely, so drastically change > many functions.
> Please comments.